.github/workflows/build.yml

```diff
@@ -15,17 +15,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-        with:
-          fetch-depth: 0
```

Comment:
What was changed since https://github.com/buildpacks/lifecycle/pull/415/files#r494993607 ?

Reply:
We don't need the history to run tests but we do need it to build and package, so I moved these lines to the `build` job https://github.com/buildpacks/lifecycle/pull/444/files#diff-5c3fa597431eda03ac3339ae6bf7f05e1a50d6fc7333679ec38e21b337cb6721R54-R55

Reply:
You're right. I missed it. Thanks for the clarification!